exporter/stackdriverexporter/README.md

```diff
@@ -0,0 +1,3 @@
+# Stackdriver Exporter
```

Comment:
Maybe we should open an issue for each of these and assign them to the various vendors/owners?

Reply:
Created issues for these